dotcom-rendering/src/components/SignInGateSelector.importable.tsx

```diff
@@ -196,6 +196,8 @@ export const SignInGateSelector = ({
 	const shouldPersonaliseComponentId = (): boolean => {
 		return personaliseSwitch && !!checkoutCompleteCookieData;
 	};
+	const personaliseSignInGateAfterCheckout =
+		switches.personaliseSignInGateAfterCheckout;
```

Comment:
Destructuring might be more common?

```suggestion
	const { personaliseSignInGateAfterCheckout } =
		switches;
```